client/components/AIAssistant.js

```diff
@@ -48,13 +48,20 @@ export const AIAssistant = ({ roomUrl }) => {
     "app-message",
     useCallback((ev) => {
       const data = ev?.data;
-      if (data?.kind === "ai-summary") {
-        setSummary(data.data);
+      if (!data) return;
+      const kind = data.kind;
+      const err = data.error;
+      if (err) {
+        playAudioError();
+      }
+      const msg = err ? err : data.data;
```

Comment:
Is `data.error` undefined or null when it's not set? In that case we can shorten this line slightly ☺️ 
```suggestion
      const msg = err ?? data.data;
```